### PR TITLE
docs: fix link of hot-reload in docs

### PR DESCRIPTION
**Pull request by LeBW**
### Description

Fix link of hot-reload in docs
### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Apache APISIX differs in the following ways:
 
 - It uses etcd to save and synchronize configurations rather than relational databases like PostgreSQL or MySQL. The real-time event notification system in etcd is easier to scale than in these alternatives. This allows APISIX to synchronize the configuration in real-time, makes the code concise and avoids a single point of failure.
 - Fully dynamic.
-- Supports [hot loading of Plugins](/docs/apisix/plugins/#hot-reload).
+- Supports [hot loading of Plugins](terminology/plugin.md#hot-reload).
```
Comment (hf400159) [inline docs/en/latest/FAQ.md:36]:
```suggestion
- Supports [hot loading of Plugins](./terminology/plugin.md#hot-reload).
```

File: docs/en/latest/FAQ.md [hunk 2]
```diff
@@ -249,7 +249,7 @@ nginx_config:
 
 All Plugins in Apache APISIX are hot reloaded.
 
-You can learn more about hot reloading of Plugins [here](https://apisix.apache.org/docs/apisix/architecture-design/plugin/#hot-reload).
+You can learn more about hot reloading of Plugins [here](terminology/plugin.md#hot-reload).
```
Comment (hf400159) [inline docs/en/latest/FAQ.md:252]:
```suggestion
You can learn more about hot reloading of Plugins [here](./terminology/plugin.md#hot-reload).
```

File: docs/zh/latest/FAQ.md [hunk 3]
```diff
@@ -252,7 +252,7 @@ nginx_config:
 
 所有的 Apache APISIX 的插件都支持热加载的方式。
 
-如果你想了解更多关于热加载的内容，请参考[热加载](../terminology/plugin#热加载)。
+如果你想了解更多关于热加载的内容，请参考[热加载](terminology/plugin.md#热加载)。
```
Comment (hf400159) [inline docs/zh/latest/FAQ.md:255]:
```suggestion
如果你想了解更多关于热加载的内容，请参考[热加载](./terminology/plugin.md#热加载)。
```

File: docs/zh/latest/FAQ.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Apache APISIX 在以下方面有所不同：
 — 它使用 etcd 来保存和同步配置，而不是使用如 PostgreSQL 或 MySQL 这类的关系数据库。etcd 中的实时事件通知系统比这些替代方案更容易扩展。这允许 APISIX 实时同步配置，使代码简洁，并避免单点故障。
 
 - 完全动态
-- 支持[热加载插件](/docs/apisix/terminology/plugin#热加载)。
+- 支持[热加载插件](terminology/plugin.md#热加载)。
```
Comment (hf400159) [inline docs/zh/latest/FAQ.md:37]:
```suggestion
- 支持[热加载插件](./terminology/plugin.md#热加载)。
```

**Review by hf400159 — COMMENTED** [review]
greate

**Comment by LeBW** [comment]
@hf400159 fixed

**Review by SylviaBABY — APPROVED** [review]
LGTM, and thanks for your contribution^^

**Review by juzhiyuan — APPROVED** [review]

**Comment by lijing-21** [comment]
Hi @LeBW  , thanks for the contribution!

Here is the Contributor T-shirt form[1], if you're interested, kindly take a look :)

[1] https://github.com/apache/apisix/blob/master/CONTRIBUTING.md#contributor-t-shirt

**Pull request merged**